<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StuAdd</class>
 <widget class="QWidget" name="StuAdd">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>150</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>150</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>150</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>楷体</family>
    <pointsize>16</pointsize>
    <italic>false</italic>
   </font>
  </property>
  <property name="windowTitle">
   <string>增加学生</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../Qrc/20240822_Mine1.qrc">
    <normaloff>:/new/prefix1/X:/PS/Photos/L_S_E/L_S_E-1_1.jpg</normaloff>:/new/prefix1/X:/PS/Photos/L_S_E/L_S_E-1_1.jpg</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="L1_Name">
       <property name="minimumSize">
        <size>
         <width>55</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>55</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>16</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>姓名:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LE1_Name">
       <property name="minimumSize">
        <size>
         <width>125</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>145</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>16</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="placeholderText">
        <string>请输入</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="L2_SN">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>16</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>SN:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LE2_SN">
       <property name="minimumSize">
        <size>
         <width>410</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>470</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>16</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="placeholderText">
        <string>请输入(总分自动计算默认保留一位小数)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_1">
     <item>
      <widget class="QLabel" name="L3_YW">
       <property name="minimumSize">
        <size>
         <width>55</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>55</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>16</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>语文:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LE3_YW">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>16</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="placeholderText">
        <string>请输入</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="L4_SX">
       <property name="minimumSize">
        <size>
         <width>55</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>55</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>16</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>数学:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LE4_SX">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>16</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="placeholderText">
        <string>请输入</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="L5_YY">
       <property name="minimumSize">
        <size>
         <width>55</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>55</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>16</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>英语:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LE5_YY">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>16</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="placeholderText">
        <string>请输入</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="L6_ZF">
       <property name="minimumSize">
        <size>
         <width>55</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>55</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>16</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>总分:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="L7_ShowZF">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>自动计算</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="PB1_Yes">
       <property name="minimumSize">
        <size>
         <width>330</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>350</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>16</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>确认</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PB2_No">
       <property name="minimumSize">
        <size>
         <width>330</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>350</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>16</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../Qrc/20240822_Mine1.qrc"/>
 </resources>
 <connections/>
</ui>
